src/index.js

```diff
@@ -3516,6 +3516,10 @@ export class ULabel {
         if (subtask == null) {
             subtask = this.state["current_subtask"];
         }
+        // No need to rebuild contianing box for image-level annotation types.
+        if (this.subtasks[subtask]["annotations"]["access"]["spatial_type"] in ["whole-image", "global"]) {
```

Comment:
Should probably use `NONSPATIAL_MODES` here instead of `["whole-image", "global"]`